server/templates/tools/map.html

```diff
@@ -34,4 +34,8 @@
 {% block footer %}
   <script src={{url_for('static', filename='map.js', t=config['VERSION'])}}></script>
   <script src="https://maps.googleapis.com/maps/api/js?key={{maps_api_key}}&libraries=places" async defer></script>
+{% endblock %}
+
+{% block subfooter_extra %}
+<button id="bq-link" class="btn-subfooter btn btn-primary" style="display: none">Big Query</button>
```

Comment:
Nit: Should be BigQuery without a space.

Should we add an action word -- Open in BigQuery?

```suggestion
<button id="bq-link" class="btn-subfooter btn btn-primary" style="display: none">BigQuery</button>
```

Reply:
done